client/hosting/overview/components/migration-overview/components/migration-pending/index.tsx

```diff
@@ -71,6 +84,15 @@ export const MigrationPending = ( { site }: { site: SiteDetails } ) => {
 
 	return (
 		<Container>
+			<ConfirmModal
+				isVisible={ isConfirmModalVisible }
+				onCancel={ () => setIsConfirmModalVisible( false ) }
+				onConfirm={ handleCancelButtonClick }
+				title={ translate( 'Are you sure?' ) }
+				text={ translate( 'Are you sure you want to cancel your migration?' ) }
+				confirmButtonLabel={ translate( 'Yes, I want to cancel my migration' ) }
```

Comment:
@fditrapani if you decide to update the button labels, please also update the test files:

https://github.com/Automattic/wp-calypso/pull/96278/files#diff-ee2b43add907db0cf49d4fcd180b4ba5e83d14b174ed2882fb67e823e81c6658R45


https://github.com/Automattic/wp-calypso/pull/96278/files#diff-ee2b43add907db0cf49d4fcd180b4ba5e83d14b174ed2882fb67e823e81c6658R58-R59